client/src/wikis/article-editor.tsx

```diff
@@ -45,6 +50,12 @@ export const ArticleEditor = ({
     .map((section) => section.category)
     .filter((cat) => !!cat);
 
+  useEffect(() => {
+    if (mode === "create" && defaultValues?.categoryKey) {
+      form.reset({ categoryKey: defaultValues.categoryKey });
```

Comment:
est-ce qu'on a pas envie de passer toutes les valeurs par défaut (même si pour l'instant y a que la `categoryKey` ? Et pourquoi on veut pas le faire en mode update ? 

genre 


```suggestion
    if (mode === "create" && defaultValues) {
      form.reset(defaultValues);
```

Reply:
Okay j'ai fais ton changement, par contre que je mette mode === "create" ou rien du tout bah ca change rien (je crois), vu que quand je fais update (edit dans la console) tout est pre remplie de toute facon. Donc autant ne rien mettre c'est plus clean comme tu l'as suggéré. Voici ce que j'ai utilisé.: 

  useEffect(() => {
    if (defaultValues) {
      form.reset(defaultValues);
    }
  }, [defaultValues, mode, form]);